### DISCUSSION TITLE
所有数据都存在一个json长期使用文件会不会太大

### DISCUSSION THREAD

**Discussion by hiforrest** [Kenshin/simpread#2767] [讨论组]
我用了三四天，标注了26篇文章，simpread_config.json文件大小为553KB。按照这个速度一个月能达到十几M，日积月累达到上百M也并非天方夜谭

**Comment by Kenshin** [comment]
哈哈，除非你标注了某些带有 base64 的图片，否则没问题的，看看我的。

![image](https://user-images.githubusercontent.com/81074/131797425-79ffd3bd-7195-460b-bba4-f1633f705b2b.png)

![image](https://user-images.githubusercontent.com/81074/131797459-63e00755-33b5-4dc8-bcca-f0032694bcb2.png)

另外，后续的版本会拆分为更好的方案。

**Reply by hiforrest** [reply]
1500多篇文章才7.8M。我的使用习惯是有用的信息都会标注（以后浏览不用打开原网页）导致标注内容比较多吧

**Reply by Kenshin** [reply]
> 1500多篇文章才7.8M。我的使用习惯是有用的信息都会标注（以后浏览的不用打开原网页）导致标注内容比较多吧

1. 不会有性能问题，我测试过 20M（再大的话，就没测试过了
2. 你可以打开你的 `simpread_config.json`  看看里面是不是有 base64 ？ 否则只是标注的信息不会很多。

**Reply by hiforrest** [reply]
我检查了没有base64。需要的话可以把文件发给你。

**Reply by Kenshin** [reply]
@hiforrest 

可以的，因为没有涉及到配置方面的问题，所以脱敏后的 `simpread_config.json ` 发到我的邮箱里面。（在我的 Github 可找到，标题加上 #2767

**Reply by hiforrest** [reply]
已发

**Reply by Kenshin** [reply]
嗨，我看了下你的稍后读，因为标注需要对原文进行定位，所以在一些奇葩的页面下会导致定位的信息过多，而你给我的配置文件按照你理解的这么大，就是因为有3篇这样文章。

确实太稀奇了，一般来说，我 1.5K 的收藏，有效标注超过 1K，只遇到两三篇这样的情况。

我会研究下「这几个有问题」的定位，看看能否完美解决它。

**Reply by hiforrest** [reply]
找到问题是好事啊，辛苦了

**Reply by Kenshin** [reply]
哈哈，没事，也不算是 Bug 但确实需要避免此问题。
